userspace/src/usbip/usbip_wudev.c

```diff
@@ -70,15 +70,24 @@ fetch_conf_desc(SOCKET sockfd, unsigned devid, char *pdesc, unsigned desc_size)
 static void
 supplement_with_interface(SOCKET sockfd, usbip_wudev_t *wudev)
 {
-	unsigned char	buf[9];
+	unsigned char	buf[18];
 
-	if (fetch_conf_desc(sockfd, wudev->devid, buf, 9) < 0) {
+	if (fetch_conf_desc(sockfd, wudev->devid, buf, 18) < 0) {
 		err("failed to adjust device descriptor with interface descriptor\n");
 		return;
 	}
 
 	wudev->bNumInterfaces = buf[4];
 
+	//initialize cls/subcls/pro for 1 interface device before importing
+	if (wudev->bNumInterfaces == 1)
```

Comment:
Please, put a brace to the same line and remove empty lines. 😄 Coding style

```
if (wudev->bNumInterfaces == 1) {
}
```

Reply:
sure, will definitely follow it. :)